src/packages/@common/components/sidebar/SidebarProfileModal.svelte

```diff
@@ -80,30 +80,26 @@
   on:mouseenter={() => (isHovered = true)}
   on:mouseleave={() => (isHovered = false)}
   on:click={(e) => {
-    showModal = !showModal;
+    showProfileModal = !showProfileModal;
   }}
 >
-  <div class="d-flex align-iems-center justify-content-center">
-    {#if isHovered && item.hoveredLogo && !item.disabled}
-      <img src={item.hoveredLogo} alt={item.heading} />
-    {:else if isRouteActive && item.selectedLogo}
-      <img src={item.selectedLogo} alt={item.heading} />
-    {:else}
-      <img src={item.defaultLogo} alt={item.heading} />
-    {/if}
-  </div>
+<div class="d-flex align-iems-center justify-content-center">
+  {#if isHovered && item.hoveredLogo && !item.disabled}
+    <img src={item.hoveredLogo} alt={item.heading} />
+  {:else if isRouteActive && item.selectedLogo}
+    <img src={item.selectedLogo} alt={item.heading} />
+  {:else}
+    <img src={item.defaultLogo} alt={item.heading} />
+  {/if}
+</div>
```

Comment:
if we are not using this component in sidebar, then remove this from sidebar directory

Reply:
Resolved